docs/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+
+## 1.0.0b1
+
+Feature:
+* Add new arguments to `aql.load_file` - `native_support_kwargs` and `enable_native_fallback`. `native_support_kwargs` is a keyword argument to be used by method involved in native support flow and `enable_native_fallback` can be used to fall back to default transfer. `aql.load_file`, by default, uses native approaches to load datasets into Google BiqQuery and Snowflake. The old behaviour can be re-enabled by setting `use_native_support=False`. [#557](https://github.com/astronomer/astro-sdk/issues/557), [#481](https://github.com/astronomer/astro-sdk/issues/481)
+* Implement fallback mechanism in case native support fails to default option with log warning for problem with native support. [#557](https://github.com/astronomer/astro-sdk/issues/557)
+* Add Example DAG for Dynamic Map Task with Astro-SDK. [#377](https://github.com/astronomer/astro-sdk/issues/377),[airflow-2.3.0](https://airflow.apache.org/blog/airflow-2.3.0/)
+
+
+Internals:
```

Comment:
As we spoke, we can name this session Community since it is probably one of the first improvements we made for outside developers.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/576/commits/850c76559f23e121934883691e089807411cc309)